docs/apache-airflow-providers-arangodb/operators/index.rst

```diff
@@ -32,15 +32,15 @@ further process the result using **result_processor** Callable as you like.
 
 An example of Listing all Documents in **students** collection can be implemented as following:
 
-.. exampleinclude:: /../../airflow/providers/arangodb/example_dags/example_arangodb.py
+.. exampleinclude:: /../../tests/system/providers/arangodb/example_arangodb.py
     :language: python
     :start-after: [START howto_aql_operator_arangodb]
     :end-before: [END howto_aql_operator_arangodb]
 
 You can also provide file template (.sql) to load query, remember path is relative to **dags/** folder, if you want to provide any other path
 please provide **template_searchpath** while creating **DAG** object,
 
-.. exampleinclude:: /../../airflow/providers/arangodb/example_dags/example_arangodb.py
+.. exampleinclude:: /../../tests/system/providers/arangodb/example_arangodb.py
     :language: python
     :start-after: [START howto_aql_operator_template_file_arangodb]
```

Comment:
```suggestion
    :dedent: 4
    :start-after: [START howto_aql_operator_template_file_arangodb]
```
Now that the `START/END` markers have been indented in the system test example DAG, this option should be added to the `exampleinclude::` directives in the docs so the code snippets are rendered properly.